tests/_helpers.js

```diff
@@ -172,11 +173,19 @@ async function testCLI(t, { fixtures = [], args, message }) {
   )
 }
 
-function runCLI({ args = [], cwd = process.cwd() }) {
+function runCLI({ args = [], cwd = process.cwd(), stdin }) {
   return new Promise((resolve) => {
-    execFile('node', [cliScript, ...args], { cwd }, (error, stdout, stderr) => {
-      resolve({ errorCode: error && error.code, stdout, stderr })
-    })
+    const cp = execFile(
+      'node',
+      [cliScript, ...args],
+      { cwd },
+      (error, stdout, stderr) => {
+        resolve({ errorCode: error && error.code, stdout, stderr })
+      },
+    )
+    if (stdin) {
+      cp.stdin.end(stdin)
```

Comment:
Can stdin pass through  third argument?

Reply:
I don't think so, I didn't see any option to pass stdin in the docs 

![image](https://user-images.githubusercontent.com/17787042/232989019-13e8084c-87f0-4672-86b9-d5821d38822b.png)